src/modules/project/project.constants.js

```diff
@@ -4,6 +4,16 @@ export const OVERWRITE = 'OVERWRITE';
 export const UPLOAD_TO_SERVER = 'UPLOAD_TO_SERVER';
 export const CANCEL = 'CANCEL';
 
+export const DEFAULT_GEOLOGIC_TYPES_DICTIONARY = [
```

Comment:
`DEFAULT_GEOLOGIC_TYPES` is already below. You should be able to use it as an array.